spec/models/question_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Question, type: :model do
+  let(:quiz) { Quiz.create!(title: "Sample Quiz", description: "This is a sample quiz") }
```

Comment:
```suggestion
  let(:quiz) { Quiz.create!(title: "Sample Quiz", description: "This is a sample quiz") }
```

You don't need to do this setup to test a model directly (and indeed it makes it massively slower as it uses the database)

Reply:
Done this now!